src/main/java/org/web3j/tx/HighThroughputTransactionManager.java

```diff
@@ -0,0 +1,127 @@
+package org.web3j.tx;
+
+import org.web3j.crypto.Credentials;
```

Comment:
I'd be grateful if you can use the following import ordering for consistency.

![screen shot 2017-02-10 at 12 31 50](https://cloud.githubusercontent.com/assets/2148266/22810441/ecf34606-ef8c-11e6-9ee1-4f0cc4cdbc10.png)